backend/src/main/java/codezap/template/service/facade/TemplateApplicationService.java

```diff
@@ -87,10 +90,19 @@ public FindAllTemplatesResponse findAllBy(
             Pageable pageable,
             Member loginMember
     ) {
-        Page<Template> templates = templateService.findAllBy(memberId, keyword, categoryId, tagIds, pageable);
+        Page<Template> templates = templateService.findAllBy(
+                memberId, keyword, categoryId, tagIds, getDefaultVisibility(memberId, loginMember), pageable
+        );
         return makeResponse(templates, (template -> likesService.isLiked(loginMember, template)));
     }
 
+    private Visibility getDefaultVisibility(Long memberId, Member loginMember) {
+        if (memberId == null || memberId.equals(loginMember.getId())) {
```

Comment:
⚡️loginMember 에게 물어볼까요~~~~~~~~~~~?
```suggestion
        if (memberId == null || loginMember.hasSameId(memberId)) {
```